### DISCUSSION TITLE
Add shop=florist to the POI selection

### DISCUSSION THREAD

**Discussion by Strubbl** [FacilMap/facilmap#238] [Ideas]
It would be nice to have a checkbox for florist shops in the POIs tab. I mean shop objects using this tag: https://wiki.openstreetmap.org/wiki/Tag:shop%3Dflorist

**Comment by cdauth** [comment]
Good suggestion. In the meantime, you can find florists by switching to custom query mode and using the query `nwr[shop=florist]`.

**Comment by cdauth** [comment]
This is now added and will be released with FacilMap 4. You can already try it out on https://beta.facilmap.org/
